app/views/widgets/show.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.partial! "widgets/widget", widget: @widget
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.